libs/service-portal/applications/src/screens/ApplicationList/ApplicationList.tsx

```diff
@@ -6,21 +6,29 @@ import {
 } from '@island.is/service-portal/core'
 import ApplicationCard from '../../components/ApplicationCard/ApplicationCard'
 import { Typography, Box, Stack } from '@island.is/island-ui/core'
-import { useListApplications } from '@island.is/service-portal/graphql'
+import { useApplicantApplications } from '@island.is/service-portal/graphql'
 import { Application } from '@island.is/application/core'
 
 const ApplicationList: ServicePortalModuleComponent = ({ userInfo }) => {
-  const { data: applications, loading, error } = useListApplications(
+  const { data: applications, loading, error } = useApplicantApplications(
     userInfo.profile.natreg,
   )
 
+  const {
+    data: assigneeApplications,
+    loading: assigneeApplicationsLoading,
+    error: assigneeApplicationsError,
+  } = useApplicantApplications(userInfo.profile.natreg)
+
   return (
     <>
       <Box marginBottom={5}>
         <Typography variant="h1">Umsóknir</Typography>
       </Box>
-      {loading && <ActionCardLoader repeat={3} />}
-      {error && (
+      {(loading || assigneeApplicationsLoading) && (
```

Comment:
I'm guess this is still a WIP since we haven't nailed how we will be handling applications whether you're the applicant or an assignee.

But as it stands now this implementation could create a state where one application list has been fetched and the other hasn't, showing both a loading message and a list at the same time

Reply:
Yeah, I forgot to remove the assigneeLoading and assigneeError in theses checks, there is a seperate loading and error for that list below.